internal/host/hostcommands/install_cmd.go

```diff
@@ -7,6 +7,7 @@ import (
 	"net"
 	"strings"
 
+	"github.com/asaskevich/govalidator"
```

Comment:
this was last released in 2021 - and not look like it been mantained in a while 

maybe it's better to use the builtin net package
https://pkg.go.dev/net/netip#Addr.Is4

```go
package main

import (
	"fmt"
	"net/netip"
)

func main() {
	addrString := "192.168.1.1"
	addr, err := netip.ParseAddr(addrString)
	if err != nil {
		fmt.Println("Error parsing")
	}
	fmt.Printf("%v", addr.Is4())
	// output: true
	fmt.Printf("%v", addr.Is6())
	// output: false

}

```